src/tool/cppwinrt/cppwinrt/settings.h

```diff
@@ -22,13 +24,14 @@ namespace xlang
         bool component_opt{};
         bool component_ignore_velocity{};
 
-        bool verbose{};
-
         std::set<std::string> include;
         std::set<std::string> exclude;
 
         meta::reader::filter projection_filter;
         meta::reader::filter component_filter;
+
+        bool fastabi{};
+        std::map<meta::reader::TypeDef, meta::reader::TypeDef> fastabi_defaults;
```

Comment:
>fastabi_defaults [](start = 63, length = 16)

nit: kind of a funny name (sounds like "default settings for fastabi")

Reply:
Yes, naming is hard. Maybe just `fastabi_cache` since the function that creates it is called `build_fastabi_cache`.